.github/workflows/rust.yml

```diff
@@ -38,10 +38,13 @@ jobs:
       run: |
         export RUSTFLAGS="-Cinstrument-coverage"
         export LLVM_PROFILE_FILE='cargo-test-%p-%m.profraw'
-        cargo nextest run --workspace --exclude examples
+        cargo nextest run --workspace
```

Comment:
We need to bring back the `--exclude examples` because creating the binaries for the examples uses a crazy amount of storage, specially on Windows.

I would instead add a new command just for examples, something like this

```
cargo check --examples
```

Reply:
added